plugins/com.google.cloud.tools.eclipse.appengine.newproject.maven/src/com/google/cloud/tools/eclipse/appengine/newproject/maven/messages.properties

```diff
@@ -9,9 +9,8 @@ LOCATION_GROUP_TEXT=Location
 JAVA_PACKAGE_LABEL=Java package:
 GENERATED_PROJECT_LOCATION=Select the directory to contain the generated project
 PROVIDE_LOCATION=Provide a location
-NONWRITABLE=Selected location {0} cannot be created.
 FILE_LOCATION=Selected location {0} is a file.
-INVALID_PATH={0} is not a directory path.
+INVALID_PATH=Cannot create directory {0} for workspace because {1}.
```

Comment:
The _because_ formatting will look weird if the reason text is mixed case.  For example:

> Cannot create directory foo/baz/bar for workspace because File Already Exists: foo/baz/bar.